templates/budget.yaml

```diff
@@ -150,7 +150,7 @@ Resources:
         - Notification:
             NotificationType: ACTUAL
             ComparisonOperator: GREATER_THAN
-            Threshold: 100
+            Threshold: 200
```

Comment:
hmm, seems odd.  is this saying you want to be notified only if the cost is 200% over the budget?

Reply:
I want to be notified when the forecasted cost passes 100%, when the actual cost passes 100%, and when the actual cost passes 200%. 

The reason for the new 200% threshold is to account for normal spikes in cost, which might happen a few times per year. When I get a notification and confirm that the spike is normal, I want to still be notified if the cost continues to grow much beyond the 100% threshold. I don't want to permanently increase the threshold because most months should be below it.